projects/packages/my-jetpack/_inc/components/connection-status-card/styles.module.scss

```diff
@@ -25,35 +33,30 @@ $myjetpack-connection-avatar-overlap: 10px;
 		.avatar {
 			width: $myjetpack-connection-avatar-width;
 			position: absolute;
-			margin-left: -( $myjetpack-connection-avatar-overlap );
+			margin-inline-start: -( $myjetpack-connection-avatar-overlap );
 			border-radius: 50%;
 		}
 
 		.line {
-			width: 67px;
-			height: 0;
-			border-top: 2px solid var( --jp-black );
-			border-top-style: dashed;
+			width: 1.875rem; // 30px
```

Comment:
Nitpick: Is there a reason this unit is rem and the others here are px?

Reply:
Yes, we want the width to scale, but not the thickness.